distribution/docs/src/main/jdocs/content/_endpoints/endpoints-intro-contents.adoc

```diff
@@ -0,0 +1,28 @@
+
+.Endpoints
```

Comment:
These section titles appear really similar to the titles above and might need some indentation as well.
![screen shot 2017-03-14 at 9 46 18 am](https://cloud.githubusercontent.com/assets/11355332/23911797/1afecff8-089b-11e7-995a-ea3f0ef9d616.png)